### ISSUE TITLE
The game crashes

### ISSUE THREAD

**Issue by Gregrs72kd** [Northfear/fallout1-ce-vita#4]
Hello everyone. Who can solve my problem? I installed all the files according to the instructions, the introductory video is played at startup, and then the game is minimized. GOG version.
Thank you in advance for your help.

**Comment by Northfear** [comment]
Either something is wrong with the data or there's something weird in fallout.cfg. What is the language of the game?
Occasional crashes might happen during skipping of movies, but it's not commonly reproducible in fallout 1.

**Comment by Gregrs72kd** [comment]
The language is English. I installed Fallout on my computer and immediately transferred the files without changes.

**Comment by Northfear** [comment]
What is the reproduction rate? 100%?

**Comment by Northfear** [comment]
Have you transferred `f1_res.ini` too btw?

**Comment by Gregrs72kd** [comment]
No, I transferred only the files that are in the instructions. Actually, for some reason, some third-party games don't start, I thought it was a problem in the files. Of course, there may be a problem in "libshacccg.suprx", SharkF00D does not work for me, I had to manually install it.

**Comment by Northfear** [comment]
Sounds like the problem is unrelated to the port itself. Some plugin problems? Let us know is you'll figure it out
